vue/src/modules/Auth/Login.vue

```diff
@@ -15,10 +14,10 @@
         <form v-on:submit.prevent="onLoginClick">
           <ValidationObserver ref="loginForm">
             <div class="auth-form">
-              <input-widget ref="usernameInputWidget" :model="model" attribute="username"/>
+              <input-widget ref="emailInputWidget" :model="model" attribute="email"/>
               <input-widget :model="model" attribute="password" type="password"/>
               <div class="d-flex align-items-center justify-content-between">
-                <button class="btn btn-primary mr-2">{{ $t('Login') }}</button>
+                <button class="btn btn-primary rounded-0 mr-2">{{ $t('Login') }}</button>
```

Comment:
@ll-bat no need to add rounded-0 class. Let the button have default rounded size.